.idea/python-airzone.iml

```diff
@@ -2,7 +2,7 @@
 <module type="PYTHON_MODULE" version="4">
   <component name="NewModuleRootManager">
     <content url="file://$MODULE_DIR$" />
-    <orderEntry type="jdk" jdkName="Python 3.7" jdkType="Python SDK" />
+    <orderEntry type="inheritedJdk" />
```

Comment:
I know it's also my fault, but we need to avoid to add the pycharm config files.
The main branch doesn't have any editor configuration.
We need to add a .idea to the .gitignore
Once the branch is finished I will avoid to merge to the main branch any editor config